lease/lessor.go

```diff
@@ -504,8 +505,8 @@ type Lease struct {
 	ttl int64 // time to live in seconds
 
 	itemSet map[LeaseItem]struct{}
-	// expiry time in unixnano
-	expiry  time.Time
+	// expiry is time when lease should expire
+	expiry  time.Duration
```

Comment:
This is the wrong type; Durations aren't deadlines. Probably have a `type monotime.Time uint64` (to indicate it's not a time based on the system time) or something?